app/assets/stylesheets/shared/form.scss

```diff
@@ -62,6 +62,18 @@
   }
 }
 
+label[required="required"],
+label[required="true"] {
```

Comment:
One of these can likely be removed after you do your thing noting required labels.

Reply:
I could see this being even larger/bolder!  But- that seems tricky to align.  So this is a start!